spec/support/features/in_person_helper.rb

```diff
@@ -138,8 +138,9 @@ def complete_address_step(_user = nil, same_address_as_id: true)
     click_idv_continue
   end
 
-  def complete_ssn_step(_user = nil)
+  def complete_ssn_step(_user = nil, tmx_status = nil)
```

Comment:
added this arg to help toggle the tmx_status. However, we don't _always_ toggle the value, hence the default nil.

Reply:
this is a great addition